master/getting-started/kubernetes/installation/calico.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Installing Calico on Kubernetes with Calico networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/calico'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+
+- If your cluster has RBAC enabled, issue the following command to 
+  configure the roles and bindings that {{site.prodname}} requires.
+
+   ```
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+   ```
+   > **Note**: You can also 
+   > [view the manifest in your browser](hosted/rbac-kdd.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+   
+## Installing {{site.prodname}} with {{site.prodname}} networking
+
+### About installing {{site.prodname}} with {{site.prodname}} networking
```

Comment:
This heading is a bit of a mouthful :)

So is the one above it. Maybe we can call is "About Installing Calico"